app/views/teaching_events/about_git_events.html.erb

```diff
@@ -61,7 +61,7 @@
         </section>
       <% else %>
         <section class="col col-720 col-space-m">
-          <p><strong>Our autumn Get Into Teaching events are now closed. Details of our spring events will be published on this page in early January 2023.</strong></p>
+          <p><strong>Our Get Into Teaching events are now closed. Details of our next events will be published on this page soon.</strong></p>
```

Comment:
```suggestion
          <p><strong>We do not have any Get Into Teaching events open for registration right now. We'll update this page with new Get Into Teaching events soon.</strong></p>
```
I'm wondering if the first sentence almost implies that they've stopped completely? would it be better to go with something a bit like this?